js/components/foia_personnel.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function displayName(foiaPersonnel) {
+  if (foiaPersonnel.name && foiaPersonnel.title) {
+    return `${foiaPersonnel.name}, ${foiaPersonnel.title}`;
+  }
+
+  return foiaPersonnel.name || foiaPersonnel.title;
```

Comment:
Do we know that one of these will always exist? Would it make sense to default to something generic instead?

Reply:
No, I would bet that in some cases there are only phone numbers or something... I don't really know what to expect... this all comes from the scrapers but we should be defensive here. Currently, this works okay, if neither are defined we render an empty `<p>`, so we could add a check to verify that the displayName is defined before rendering it. https://github.com/18F/beta.foia.gov/blob/605ca185cb0172dfcc2378d90c5795c056c2547a/js/components/foia_personnel.jsx#L21

When parsing the FOIA personnel, I do set a default title in most cases, except for `foia_misc` because I'm not sure what that field is.

Reply:
... But I don't think setting a default is right, not here at least because the title could be anything really.